sys/linux/vnet.txt

```diff
@@ -1108,14 +1108,62 @@ icmp_address_reply_packet {
 	mask	int32be
 } [packed]
 
-icmp_types = ICMP_ECHOREPLY, ICMP_DEST_UNREACH, ICMP_SOURCE_QUENCH, ICMP_REDIRECT, ICMP_ECHO, ICMP_TIME_EXCEEDED, ICMP_PARAMETERPROB, ICMP_TIMESTAMP, ICMP_TIMESTAMPREPLY, ICMP_INFO_REQUEST, ICMP_INFO_REPLY, ICMP_ADDRESS, ICMP_ADDRESSREPLY
+icmp_ext_echo_packet {
+	type		const[ICMP_EXT_ECHO, int8]
+	code		const[0, int8]
+	csum		csum[parent, inet, int16be]
+	id		int16be
+	seq_num		int16be
+	icmp_ext_hdr	icmp_ext_hdr
+	iio		icmp_ext_echo_iio
+	data		array[int8]
```

Comment:
Does the packet include any arbitrary payload beyond what's described above?

The question is related to both request and reply structures.